engine/utils/cortex_utils.h

```diff
@@ -34,8 +34,10 @@ constexpr static auto kTensorrtLlmPath = "/engines/cortex.tensorrt-llm";
 
 inline std::string models_folder = "./models";
 inline std::string logs_folder = "./logs";
-inline std::string logs_base_name = "./logs/cortex";
+inline std::string logs_base_name = "./logs/cortex.log";
+inline std::string logs_cli_base_name = "./logs/cortex-cli.log";
 inline size_t log_file_size_limit = 20000000;  // ~20 mb
+inline size_t log_file_max_lines = 100000;     // 100,000 lines
```

Comment:
@nguyenhoangthuan99 I think we should have this as a variable from `.cortexrc`

Reply:
I'll move `log_file_max_lines` to read from .cortexrc